apps/generator/lib/parser.js

```diff
@@ -143,3 +143,5 @@ function canReadFn(uri, canRead) {
   }
   return false;
 }
+
+module.exports.convertOldOptionsToNew = convertOldOptionsToNew;
```

Comment:
since you expose it outside now, please makre it properly as private usind jsdoc